packages/core/src/MachineNode.ts

```diff
@@ -38,10 +38,10 @@ export const WILDCARD = '*';
 
 const createDefaultOptions = <TContext>(
   context: TContext
-): MachineOptions<TContext, any> => ({
+): MachineImplementations<TContext, any> => ({
   actions: {},
   guards: {},
-  behaviors: {},
+  actors: {},
   delays: {},
   context
```

Comment:
q: if we are naming those as implementations, should `context` be part of it?

Reply:
Not sure, still thinking about whether this is a good idea.

Reply:
Created a card so we can decide about this later - https://github.com/davidkpiano/xstate/projects/1#card-53045090